### ISSUE TITLE
[BUG] Can't delete videos in multi-user mode

### ISSUE THREAD

**Issue by 1RandomDev** [Tzahi12345/YoutubeDL-Material#487]
**Describe the bug**
When I try to delete a video or song in multi-user mode nothing happens. In the logs there's this exception:
```
ReferenceError: usersFileFolder is not defined
  at Object.exports.deleteFile (/app/db.js:489:47)
  at async /app/app.js:1615:18
```

**To Reproduce**
1. Click on the 3 dots of a video and then 'Delete'
2. Look at the logs

**Expected behavior**
The video gets deleted and removed from the archive file.

**Screenshots**
It's obvious

**Environment**
- YoutubeDL-Material version: v4.2 (Commit 88cc8d0 )
- Docker tag: nightly 

**Additional context**
-/-

**Comment by jimbobcraig** [comment]
I can confirm that I have this issue too.

`ReferenceError: usersFileFolder is not defined
    at async /app/app.js:1615:18
    at Object.exports.deleteFile (/app/db.js:489:47)
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received null
    at validateString (internal/validators.js:124:11)
    at Object.join (path.js:1148:7)
    at collectInfo (/app/downloader.js:190:32)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) {
  code: 'ERR_INVALID_ARG_TYPE'`

**Comment by Tzahi12345** [comment]
Hi @1RandomDev @jimbobcraig I believe I fixed this in #592! Thanks for the report :)
